qiskit_machine_learning/algorithms/regressors/qsvr.py

```diff
@@ -0,0 +1,42 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Quantum Support Vector Regressor"""
+
+from typing import Union
+from sklearn.svm import SVR
+
+from qiskit import QuantumCircuit
+from qiskit.providers import Backend, BaseBackend
+from qiskit.aqua import QuantumInstance
+from qiskit.ml.algorithms import QuantumKernel
+
+class QSVR(SVR):
+    r"""Quantum Support Vector Regressor.
+
+        **Example**
+        .. code-block::
+            qsvr = QSVR(feature_map=map, quantum_instance=backend)
+            qsvr.fit(sample_train,label_train)
+            qsvr.predict(sample_test)
+    """
+
+    def __init__(self,
```

Comment:
Similar comments to the QSVC in terms of docstrings, params and consistency with setter/getters etc